README.md

```diff
@@ -49,6 +49,21 @@ Add `pod 'Yams'` to your `Podfile`.
 
 Add `github "jpsim/Yams"` to your `Cartfile`.
 
+### Bazel
+
+In your WORKSPACE file
+
+```WORKSPACE
+YAMS_GIT_SHA = "SOME_SHA"
+http_archive(
+    name = "com_github_jpsim_yams",
+    urls = [
+        "https://github.com/jpsim/Yams/archive/%s.zip" % YAMS_GIT_SHA
+    ],
+    strip_prefix = "yams-%s" % YAMS_GIT_SHA
+)
```

Comment:
Do we want to recommend using `sha256` as well?

Reply:
How do you handle this? Should we put a dummy value in here?

Reply:
I guess bazel will warn them about it, no need to mention it.